src/ZoneServer/World/Maps/Pathfinder.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Melia.Shared.World;
+using Melia.Shared.Game.Const;
+using Yggdrasil.Util;
+
+namespace Melia.Zone.World.Maps
+{
+	public class Pathfinder
+	{
+		private readonly int _maxNodeExpand = 500;
```

Comment:
There's nothing really wrong about these "consts", including the dictionary, though it would be more appropriate to make them `readonly static` and `PascalCase`, as they never change.